engine/crates/engine-v2/schema/src/builder/graph.rs

```diff
@@ -74,10 +73,10 @@ impl<'a> GraphBuilder<'a> {
     }
 
     fn ingest_config(&mut self, config: &mut Config, graph: &mut FederatedGraph) {
+        self.ingest_enums(config, graph);
         self.ingest_input_values(config, graph);
         self.ingest_input_objects(config, graph);
         self.ingest_unions(config, graph);
-        self.ingest_enums(config, graph);
```

Comment:
Necessary because the IdMap needs to be populated for the id conversion in ingest_input_values.